rate_counter/templates/tables_list.html

```diff
@@ -7,16 +7,27 @@
 {% endblock title %}
 
 {% block content %}
+{% if request.user.is_authenticated %}
+    <div class='col-sm-6 col-sm-offset-3'>
+        <p><a href="{% url 'rate:create'%}" class="btn btn-default" role="button">Create New Table</a></p>
+        {%for object in tables_list%}
+            {% if object.user = request.user%}
+                    <a href ="{% url 'rate:table_detail' object.id %}"><h2>{{object.name}}</h2></a>
+                    {% if object.user.get_full_name %}
+                        <p> Author: {{ obj.user.get_full_name }}</p>
+                    {% else %}
+                        <p> Author: {{ object.user }}</p>
+                    {% endif %}
+            {% endif %}
+        {% endfor %}
+    </div>
+{% else %}
 <div class='col-sm-6 col-sm-offset-3'>
-{%for object in tables_list%}
-    <a href ="{% url 'rate:table_detail' object.id %}"><h2>{{object.name}}</h2></a>
-    {% if object.user.get_full_name %}
-        <p> Author: {{ obj.user.get_full_name }}</p>
-    {% else %}
-        <p> Author: {{ object.user }}</p>
-    {% endif %}
-{% endfor %}
-    <br><br>
-    <p><a href="{% url 'rate:create'%}" class="btn btn-default" role="button">Create New Table</a></p>
+    <h1>
+        <p><a href="{% url 'login' %}"  role="link">Login</a></p>
```

Comment:
Those links are displayed on a page in a "middle of nowhere" :)

If possible make some box for them